Riot/SupportingFiles/Info.plist

```diff
@@ -91,5 +91,17 @@
 	<string>${PRODUCT_NAME}-Defaults</string>
 	<key>applicationGroupIdentifier</key>
 	<string>$(APPLICATION_GROUP_IDENTIFIER)</string>
+	<key>baseBundleIdentifier</key>
+	<string>$(BASE_BUNDLE_IDENTIFIER)</string>
+	<key>keychainAccessGroup</key>
+	<string>$(KEYCHAIN_ACCESS_GROUP)</string>
+	<key>pushKitAppIdProd</key>
```

Comment:
Can we avoid to put all push cert name vars in a plist?

We can instead build them directly in BuildSettings like:
````
    static var pushKitAppIdProd: String {
        self.baseBundleIdentifier + ".ios.voip.prod"
    }
````

Thus, customisation will stay in one place (under `/Config`)